os/tools/mkconfig.c

```diff
@@ -203,6 +203,10 @@ int main(int argc, char **argv, char **envp)
 	printf("#ifndef CONFIG_KMM_REGIONS\n");
 	printf("# define CONFIG_KMM_REGIONS 1\n");
 	printf("#endif\n\n");
+	printf("/* CONFIG_APP_HEAP_INDEX must be defined always */\n\n");
```

Comment:
We can add this in libelf_addrenv.c like below
```
#ifndef CONFIG_APP_HEAP_INDEX
#define CONFIG_APP_HEAP_INDEX 0
#endif
```

Reply:
Done